hoomd/hpmc/update.py

```diff
@@ -490,72 +490,56 @@ def N(self):  # noqa: N802 - allow N as a function name
         return N_dict
 
 
-class remove_drift:  # noqa - will be rewritten for v3
+class RemoveDrift(Updater):
     """Remove the center of mass drift from a system restrained on a lattice.
 
     Args:
-        mc (:py:mod:`hoomd.hpmc.integrate`): MC integrator.
-        external_lattice (:py:class:`hoomd.hpmc.field.lattice_field`): lattice
-          field where the lattice is defined.
-        period (int): the period to call the updater
-
-    The command hpmc.update.remove_drift sets up an updater that removes the
-    center of mass drift of a system every period timesteps,
+        ref_positions (`list` [`tuple` [`float`, `float`]], **required**) - the
```

Comment:
This presumes 2D currently since the list says two floats. I would prefer to see this make a NumPy array.